udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step2DescribeOrganization.vue

```diff
@@ -0,0 +1,369 @@
+                :hasError="fieldHasError('business_number_id')"
+                :hasWarning="fieldHasWarning('business_number_id')"
+                :errorText="t('This SIRET is not valid')"
+              />
+            </LinkedToAccordion>
+            <div v-if="checkOrga.exists !== null" class="fr-col fr-mx-2v fr-mb-2v bg-contrast-grey text-align-center">
+              <div v-if="checkOrga.exists">
+                <p>{{ t('The SIRET n° {number} is matching', { number: organization.business_number_id }) }}</p>
+                <p>{{ checkOrga.name }}</p>
+                <p v-if="checkOrga.isPublicService">
+                  <span class="fr-icon-bank-line" aria-hidden="true"></span>
+                  {{ t('Public Service') }}
+                </p>
+              </div>
+              <div v-else>
+                <p>{{ t('No organization found matching this SIRET on annuaire-entreprises.data.gouv.fr') }}</p>
+              </div>
+            </div>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addAcronymAccordionId"
+            >
+            <InputGroup
+                data-testid="acronymInput"
+                :aria-describedby="addAcronymAccordionId"
+                :label="t('Acronym')"
+                v-model="organization.acronym"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addDescriptionAccordionId"
+              @blur="vWarning$.description.$touch"
+            >
+            <InputGroup
+                data-testid="descriptionInput"
+                :aria-describedby="addDescriptionAccordionId"
+                :label="t('Description')"
+                :required="true"
+                type="textarea"
+                v-model="organization.description"
+                :hasError="fieldHasError('description')"
+                :hasWarning="fieldHasWarning('description')"
+                :errorText="getErrorText('description')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addWebsiteAccordionId"
+            >
+            <InputGroup
+                data-testid="websiteInput"
+                :aria-describedby="addDescriptionAccordionId"
+                :label="t('Website')"
+                type="url"
+                v-model="organization.url"
+                :hasError="fieldHasError('url')"
+                :hasWarning="fieldHasWarning('url')"
+                :errorText="getErrorText('url')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addLogoAccordionId"
+              @blur="vWarning$.acronym.$touch"
+            >
+              <p>{{ t('Logo') }}</p>
+              <UploadGroup
```

Comment:
Later on, we may wan to apply a required="true" behavior on logo? It did not appear as mandatory on Figma though